Specs/DateTime/Dutch/TimeExtractor.json

```diff
@@ -9,7 +9,6 @@
         "Length": 5
       }
     ],
-    "NotSupported": "dotNet",
```

Comment:
we built the current PR on top of the previous one because it needed the changes introduced there (I think we have already used this approach in the past).
But the commit "[.NET] Dutch DateTime TimePeriod support" contains only the changes related to TimePeriod.